.rubocop_ruby.yml

```diff
@@ -13,6 +13,8 @@ AllCops:
     - "**/Gemfile"
     - "**/Rakefile"
   Exclude:
+    - "**/node_modules/**/*"
+    - "bin/**"
```

Comment:
The main ideea in disabling the Bin folder is that most of the files there are not created or maintained by us... 

For instance:
- bin/bundle
- bin/rails 
- bin/rake
- bin/rspec 

Also, the only file that we have there, is using some global variables, and i would not go to a refactor to add a parameter to 3 methods, just because the 3rd child needs it.

Reply:
Isn't the `bin/changelog_generator` script something that is specific to Decidim and one that we control?

I think we should keep these under Rubocop too but disable the `Style/GlobalVars` rule as it is currently to avoid the violation.

Reply:
And it seems `bin/rails` is also a Decidim custom script to make it easier to work with the development app from the root.

Reply:
Ah, seems like you already took care of this at https://github.com/decidim/decidim/pull/9231/commits/b96cbd54f7880355ada71c314029115a2607cc59.

Thanks! I'll have a look.